.prettierrc.yml

```diff
@@ -5,4 +5,4 @@ printWidth: 88
 proseWrap: always
 semi: true
 trailingComma: "es5"
-xmlWhitespaceSensitivity: "ignore"
+xmlWhitespaceSensitivity: "strict"
```

Comment:
This shouldn't be `ignore`?

Reply:
This was updated by copier from https://github.com/OCA/oca-addons-repo-template/blob/master/.prettierrc.yml#L8

Should I change it back?

Reply:
This has proven to be problematic AFAIK. cc @Yajo

Reply:
Ok, I will change it back, then. 
I was worried it was going to cause trouble in any `copier update` as the upstream is `strict`, but it doesn't, so no problem.

Reply:
The problematic one was "ignore", so this update was OK. I'll fix it.